### ISSUE TITLE
Add `.help` command to the CLI

### ISSUE THREAD

**Issue by penberg** [penberg/limbo#124]
We currently just get this:

```
Welcome to Limbo SQL shell!
> .help
Unknown command: .help
Available commands:
  .schema <table_name> - Display the schema for a specific table
  .opcodes             - Display all the opcodes defined by the virtual machine
```

**Comment by JoanFM** [comment]
My humble intend to contribute!

#130

**Comment by penberg** [comment]
Fixed by e20028becca141f1246119b916c17c28c3668749
